src/__experimental__/components/checkbox/checkbox.style.js

```diff
@@ -0,0 +1,187 @@
+
+    ${HiddenCheckableInputStyle},
+    svg {
+      height: 16px;
+      position: absolute;
+      padding: 1px;
+    }
+
+    ${StyledCheckableInput},
+    ${HiddenCheckableInputStyle},
+    svg {
+      box-sizing: border-box;
+      width: 16px;
+    }
+
+    ${HiddenCheckableInputStyle}:not([disabled]) {
+      &:focus + svg,
+      &:hover + svg {
+        outline: solid 3px ${theme.colors.focus};
+      }
+    }
+
+    ${LabelStyle} {
+      padding: 0 6px;
+      text-align: ${({ labelAlign }) => labelAlign};
+      width: auto;
+
+      & ${StyledHelp} {
+        color: ${theme.help.color};
+        vertical-align: bottom;
+
+        &:hover, &:focus {
+          color: ${theme.text.color};
+        }
+      }
+    }
+
+    label:hover + ${StyledCheckableInput} ${HiddenCheckableInputStyle}:not([disabled]) + svg {
+      outline: solid 3px ${theme.colors.focus};
+    }
+
+    ${FieldHelpStyle} {
+      margin-left: 16px;
+      margin-top: 0;
+      padding-left: 6px;
+    }
+
+    ${() => size === 'large' && css`
+      ${StyledCheckableInput},
+      ${HiddenCheckableInputStyle},
+      ${HiddenCheckableInputStyle} + svg {
+        height: 24px;
+        padding: 2px;
+        width: 24px;
+      }
+
+      ${FieldHelpStyle} {
+        margin-left: 24px;
+      }
+
+      ${FieldHelpStyle},
+      ${LabelStyle} {
+        padding-left: 8px;
+      }
+
+      ${fieldHelpInline && `${FieldHelpStyle},`}
+      ${LabelStyle} {
+        padding-top: 4px;
+        padding-bottom: 4px;
+      }
+    `}
+
+    ${() => checked && css`
+      svg path {
+        fill: ${theme.colors.primary};
+      }
+    `}
+
+    ${() => disabled && css`
+      ${LabelStyle} {
+        &, & ${StyledHelp} {
+          color: ${theme.disabled.disabled};
+        }
+      }
+
+      svg {
+        background-color: ${theme.disabled.input};
+        border: 1px solid ${theme.disabled.border};
+      }
+
+      svg path { fill: ${() => (checked ? theme.disabled.border : theme.disabled.input)}; }
+
+      ${HiddenCheckableInputStyle},
+      svg,
+      ${LabelStyle} {
+        &:hover, &:focus {
+          outline: none;
+          cursor: not-allowed;
+        }
+      }
+    `}
+
+    ${() => error && css`
+      svg {
+        border: 1px solid ${theme.colors.error};
+      }
+    `}
+
+    ${() => fieldHelpInline && css`
+      ${FieldHelpStyle} {
+        display: inline;
+        margin: 0;
+        padding-left: 0;
+        width: auto;
+      }
+    `}
+
+    ${() => inputWidth !== 0 && css`
+      ${StyledCheckableInput} {
+        width: ${inputWidth}%;
+      }
+
+      ${FieldHelpStyle} {
+        ${reverse ? 'margin-right' : 'margin-left'}: ${inputWidth}%;
+      }
+    `}
+
+    ${({ labelWidth }) => labelWidth !== 0 && css`
+      ${LabelStyle} {
+        width: ${labelWidth}%;
+      }
+    `}
+
+    ${() => reverse && css`
+      ${FieldHelpStyle} {
+        margin-left: 0;
+      }
+    `}
+
+    ${() => reverse && fieldHelpInline && css`
```

Comment:
please remove all unnecessary arrow functions and styled-components `css` methods